examples/v2/cloudbuild/README.md

```diff
@@ -0,0 +1,31 @@
+# Cloud Container Builder Example
+
+## Overview
+
+This is a [Google Cloud Deployment
+Manager](https://cloud.google.com/deployment-manager/overview) template that
+creates a Cloud Container Builder build.
+
+This sample uses the undocumented DM Actions feature (estimated to come out
+in Q4 2017) to call the Cloud Container Builder projects.builds.create API.
+This doesn't create a DM resource like you would normally see, it just calls
+the API as specified in the metadat.runtimePolicy section. This section may
+contain the values UPDATE\_ALWAYS (call the API for create or update changes
+in the deployment), CREATE (only call on create), UPDATE\_ON\_CHANGE (call
+when the action changes), DELETE (call on deletes).
+
+
+## Deploy the template
+
+Use `cloudbuild.yaml` to deploy this example template. You need to ensure
+the Cloud Container Builder service account has read access to DM for the sample.
+More information can be found
+[here](https://cloud.google.com/container-builder/docs/how-to/service-account-permissions).
```

Comment:
Also enable the api https://pantheon.corp.google.com/apis/library/cloudbuild.googleapis.com/ ? I think it's obvious once you get the error message (or if you're an experienced GCP user), but might be good to make our examples as explicit as possible?

Reply:
Hmmm I'm torn on how much detail to give for these, but I don't see too much harm in it.

Done.

Reply:
Yeah I gotchu. Thanks though